src/frontend/apps/e2e/__tests__/app-impress/common.ts

```diff
@@ -91,6 +92,18 @@ export const addNewMember = async (
   return users[index].email;
 };
 
+export const getGridRow = async (page: Page, title: string) => {
+  const docsGrid = page.getByRole('grid');
+  await expect(docsGrid).toBeVisible();
+  await expect(docsGrid.getByTestId('grid-loader')).toBeHidden();
+
+  const rows = docsGrid.getByRole('row');
+
+  return rows.filter({
+    hasText: title,
+  });
```

Comment:
To avoid going further, I would test if the rose is well defined before returning it. WDYT ?